resources/views/components/user/profile/primary-meta.blade.php

```diff
@@ -55,8 +55,10 @@ class="rounded-sm h-[64px] w-[64px] sm:max-h-[128px] sm:max-w-[128px] sm:min-w-[
 
         {{-- Motto --}}
         @if (!empty($userMassData['Motto']))
-            <div class="rounded bg-box-bg px-2 py-1 max-w-fit italic text-2xs hyphens-auto mb-3">
-                <p style="word-break: break-word;">{{ $userMassData['Motto'] }}</p>
+            <div x-data="{}" class="rounded bg-box-bg px-2 py-1 max-w-fit italic text-2xs hyphens-auto mb-3">
+                <p x-linkify class="[&>a]:text-text [&>a]:transition-colors [&>a]:duration-700 [&>a]:ease-out" style="word-break: break-word;">
+                    {{ $userMassData['Motto'] }}
+                </p>
```

Comment:
It's probably a limitation of the library, but it's trying to linkify part of "file.mp3".
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/0d5c0f93-5182-49a5-97da-7c89320f3323)

Reply:
Also oddly linkifying this: 
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/5f2ef84a-5976-4233-b31e-4045a359188a)

Reply:
I've updated the directive to exclude *.mp and *.ar top level domains.